src/components/categories/MainCategory.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const MainCategory = () => (
+  <div>
+    Under construction
+  </div>
+);
+
```

Comment:
- [ ] from your response I conclude you understand what the previous CR meant when saying `Everything looks fine, but as per the requirements, you should display only a button with the text "Check status"`, please make the changes before requesting another review.